NEWS

```diff
@@ -11,6 +11,241 @@
 ### Features:
 ### Bugfixes:
 
+## 1.18.0-rc1 (November 21, 2024)
+### Feature
+#### Architecture
+ * Added CPU Model for MI300A
+ * Added Fujitsu ARM specific values to ucx.conf
+#### AZP
+ * Improved repository setup when running CI
+ * Improved test_fuzzy reporting
+ * Updated to CUDA version 12.5.1 for Fabric memory tests
+ * Improved active IB HCA detection on CI
+ * Added DRP release pipeline
+ * Removed io_demo tests for ConnectX-4
+ * Added JUCX tests on ARM
+ * Improved verification and release processes
+ * Added wire-compat testing with v1.17.x
+ * Improved reports with added server-side log prints
+ * Updated AZP images from private repository
```

Comment:
pls do not mention changes related to our internal testing/CI infra

Reply:
wbf